nowledge-mem-codex-plugin/scripts/install_hooks.py

```
@@ -0,0 +1,105 @@
+
+def install_runtime_hook() -> None:
+    HOOKS_DIR.mkdir(parents=True, exist_ok=True)
+    shutil.copy2(SOURCE_HOOK, INSTALLED_HOOK)
+    mode = INSTALLED_HOOK.stat().st_mode
+    INSTALLED_HOOK.chmod(mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+
+
+def merge_hooks_json() -> None:
+    if GLOBAL_HOOKS_FILE.exists():
+        hooks_doc = load_json(GLOBAL_HOOKS_FILE)
+        backup = GLOBAL_HOOKS_FILE.with_suffix(f".json.bak")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)